pymtl3/passes/rtlir/behavioral/BehavioralRTLIRTypeCheckL3Pass.py

```diff
@@ -97,7 +96,7 @@ def visit_StructInst( s, node ):
         "BitStruct {} has {} fields but only {} arguments are given!". \
             format(cls.__name__, len(all_properties), len(node.values)) )
 
-    all_types = zip( node.values, all_properties )
+    all_types = list(zip( node.values, all_properties ))
```

Comment:
No need for this intermediate list when we're just going to iterate over it anyway; and leaving it out is more efficient.